LICENSE.md

```diff
@@ -3,7 +3,7 @@ If you do not agree to these terms, do not access Particular Software code.
 
 Your license to Particular Software source code and/or binaries is governed by the Reciprocal Public License 1.5 (RPL1.5) license as described here: 
 
-https://www.opensource.org/licenses/rpl1.5.txt
+https://opensource.org/license/rpl-1-5/
```

Comment:
You can't change this file here. You have to change it in [RepoStandards](https://github.com/Particular/RepoStandards/blob/master/LICENSE.md)

Reply:
Will do as a separate PR